codecov.yml

```
@@ -0,0 +1,13 @@
+# Customizations to codecov for c-t-go repo. This will be merged into
+# the team / default codecov yaml file.
+#
+# Validate changes with:
+# curl --data-binary @codecov.yml https://codecov.io/validate
+
+# Exclude code that's for testing, demos or utilities that aren't really
+# part of production releases.
+ignore:
+  - "**/mock_*.go"
+  - "**/testonly"
+  - "trillian/integration/**/*"
```

Can you try this instead please?
```suggestion
  - "trillian/integration/**"
```

---

Can do.

---

That seems to have altered the coverage.